src/containers/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -133,6 +144,14 @@ const Breadcrumbs = () => {
   return (
     <Styled.Wrapper>
       <Styled.Crumbtainer>
+        {uriDisplay && localUri && (
+          <Button
+            icon="edit"
+            style={{ opacity: editMode ? 0 : 1, width: editMode ? 0 : 'auto', borderRadius: '4px 0 0 4px'  }}
+            onClick={() => setEditMode(true)}
+            variant="tonal"
+          />
+        )}
         <Styled.CrumbsForm onSubmit={goThere}>
           <label data-value={inputValue}>
             <InputText
```

Comment:
Could we please have a button at the end here so that it's more obvious that clicking/hitting enter will navigate to the URI. Maybe `arrow_forward` would work?

Also make sure there is border radius 4px on the left hand side of the input and right hand side of the new button.

![image](https://github.com/user-attachments/assets/330ed665-52ca-4dfa-aba9-2eac07bc8a6b)